### ISSUE TITLE
Django 1.11 compatibility

### ISSUE THREAD

**Issue by barseghyanartur** [SmileyChris/easy-thumbnails#442]
At the moment it breaks on Django==1.11a1.

easy-thumbnails==2.3

Trace:

```sh
Traceback (most recent call last):
  File "./runtests.py", line 16, in <module>
    sys.exit(main())
  File "./runtests.py", line 12, in main
    return pytest.main()
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/config.py", line 47, in main
    config = _prepareconfig(args, plugins)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/config.py", line 132, in _prepareconfig
    pluginmanager=pluginmanager, args=args)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 724, in __call__
    return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 338, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 333, in <lambda>
    _MultiCall(methods, kwargs, hook.spec_opts).execute()
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 595, in execute
    return _wrapped_call(hook_impl.function(*args), self.execute)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 249, in _wrapped_call
    wrap_controller.send(call_outcome)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/helpconfig.py", line 32, in pytest_cmdline_parse
    config = outcome.get_result()
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 279, in get_result
    _reraise(*ex)  # noqa
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 264, in __init__
    self.result = func()
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 596, in execute
    res = hook_impl.function(*args)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/config.py", line 882, in pytest_cmdline_parse
    self.parse(args)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/config.py", line 1032, in parse
    self._preparse(args, addopts=addopts)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/config.py", line 1003, in _preparse
    args=args, parser=self._parser)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 724, in __call__
    return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 338, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 333, in <lambda>
    _MultiCall(methods, kwargs, hook.spec_opts).execute()
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 595, in execute
    return _wrapped_call(hook_impl.function(*args), self.execute)
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 253, in _wrapped_call
    return call_outcome.get_result()
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 279, in get_result
    _reraise(*ex)  # noqa
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 264, in __init__
    self.result = func()
  File "/home/user/env/local/lib/python2.7/site-packages/_pytest/vendored_packages/pluggy.py", line 596, in execute
    res = hook_impl.function(*args)
  File "/home/user/env/local/lib/python2.7/site-packages/pytest_django/plugin.py", line 238, in pytest_load_initial_conftests
    _setup_django()
  File "/home/user/env/local/lib/python2.7/site-packages/pytest_django/plugin.py", line 134, in _setup_django
    django.setup()
  File "/home/user/env/local/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/user/env/local/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models()
  File "/home/user/env/local/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/user/env/local/lib/python2.7/site-packages/easy_thumbnails/models.py", line 6, in <module>
    from easy_thumbnails import utils, signal_handlers
  File "/home/user/env/local/lib/python2.7/site-packages/easy_thumbnails/utils.py", line 15, in <module>
    from easy_thumbnails.conf import settings
  File "/home/user/env/local/lib/python2.7/site-packages/easy_thumbnails/conf.py", line 334, in <module>
    settings = Settings()
  File "/home/user/env/local/lib/python2.7/site-packages/easy_thumbnails/conf.py", line 21, in __init__
    super(AppSettings, self).__init__(*args, **kwargs)
TypeError: __init__() takes exactly 2 arguments (1 given)
```

**Comment by SmileyChris** [comment]
This was fixed in a4467ee441920bde27c5c65c490d5d5be2599c0f

**Comment by SmileyChris** [comment]
This was fixed in a4467ee4
